tribits/doc/build_ref/TribitsBuildReferenceBody.rst

```diff
@@ -297,17 +299,27 @@ b) Create a ``*.cmake`` file and point to it [Most Recommended].
   set with ``set(<varName> <val> CACHE <TYPE> ...)`` will impact the
   configuration.
 
-  5) Cache variables force set with ``set(<varName> <val> CACHE <TYPE> "<doc>"
-  FORCE)`` in a ``<frag>.cmake`` file pulled in with ``-C <frag>.cmake`` will
-  **NOT** override a cache variable ``-D <varName>=<val2>`` passed in on the
-  command-line.  But such statements **WILL** override other cache vars set
-  with non-force statements ``set(<varName> <val1> CACHE <TYPE> "<doc>")`` in
-  the same or previously read ``-C <frag2>.cmake`` files included before the
-  file ``-C <frag>.cmake`` on the CMake command-line.  Alternatively, if the
-  file is pulled in with ``-D <Project>_CONFIGURE_OPTIONS_FILE=<frag>.cmake``,
-  then a ``set(<varName> <val> CACHE <TYPE> "<doc>" FORCE)`` statement in a
-  ``<frag>.cmake`` **WILL** override a cache variable passed in on the
-  command-line ``-D <varName>=<val2>``.
+  5) Cache variables forced set with ``set(<varName> <val> CACHE <TYPE>
+  "<doc>" FORCE)`` in a ``<frag>.cmake`` file pulled in with ``-C
+  <frag>.cmake`` will only override a cache variable ``-D<varName>=<val2>``
+  passed on the command-line if the ``-C <frag>.cmake`` argument comes
+  **after** the ``-D<varName>=<val2>`` argument (i.e. ``cmake
+  -D<varName>=<val2> -C <frag>.cmake``).  Otherwise, if the order of the
+  ``-D`` and ``-C`` arguments is reversed (i.e. ``cmake -C <frag>.cmake
+  -D<varName>=<val2>``) then the forced set() statement **WILL NOT** override
+  the cache var set on the command-line with ``-D<varName>=<val2>``.  However,
+  note that a forced set() statement **WILL** override other cache vars set
```

Comment:
Did you intend to monospace `set()` here? And in a few other places as well.

Reply:
> Did you intend to monospace `set()` here? And in a few other places as well.

Okay, I will do that since it is a CMake command.

Reply:
This is addressed in commit https://github.com/TriBITSPub/TriBITS/pull/525/commits/134f73ebf447141c53258eb0cc64ee14cf3e97d7 that I just pushed.

I will go ahead and merge now.